integration-tests/utils/integration-utils.js

```diff
@@ -29,7 +29,7 @@ const findIframe = async page => {
   return hasFrame() || await waitForFn({ fn: hasFrame })
 }
 
-const findInGrid = (page, textContains, options) => {
+const findInGrid = (page, textContains, options = { visible: true }) => {
```

Comment:
This default value will only be used if no options argument is passed to the function. For example, if `{ timeout: 1000 }` is passed as options, `visible: true` will not be applied. Would something like [`_.defaults`](https://lodash.com/docs/4.17.15#defaults) be preferable here?

Reply:
👍🏻👍🏻